tests/main/presenters/test_search_presenters.py

```diff
@@ -301,7 +302,7 @@ def test_instance_has_correct_filter_groups_for_iaas(self):
 
 
```

Comment:
We could do with better test coverage for the `build_lots_and_categories_link_tree`.
It's really heavily relied on throughout the repo and theres no test that shows exactly what its output should look like.

I still can't really tell what it should return. There's no example in the docstring or the tests unless I've missed something.

Reply:
Have added some tests that show the output of `build_lots_and_categories` which shows what it should be outputting at various levels. There are probably more test cases that should be added, but a lot of it is tested implicitly by the tests on the view.